src/reducers/authReducer.js

```diff
@@ -0,0 +1,21 @@
+export const authReducer=(state,action)=>{
+    switch(action.type){
+        case "SIGNUP_USER":
+        case "LOGIN_USER":
+            console.log(action)
+            return{
```

Comment:
This console can be removed..if not required..

Reply:
Got it.